scripts/ci/ipxe-run.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+if [[ $# -ne 2 ]]; then
+    echo "Usage: $0 <dts-base-image-path> <bzImage-path>"
+    exit 1
+fi
+
+DTS_IMAGE_PATH=$1
+DTS_IMAGE_FILENAME=$( basename $DTS_IMAGE_PATH )
+BZ_IMAGE_PATH=$2
+BZ_IMAGE_FILENAME=$( basename $BZ_IMAGE_PATH )
+IPXE_PATH=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)/../../ipxe
+
+mkdir -p $IPXE_PATH
+ln -sf $DTS_IMAGE_PATH $IPXE_PATH/$DTS_IMAGE_FILENAME
+ln -sf $BZ_IMAGE_PATH $IPXE_PATH/$BZ_IMAGE_FILENAME
```

Comment:
```suggestion
ln -srf $DTS_IMAGE_PATH $IPXE_PATH/$DTS_IMAGE_FILENAME
ln -srf $BZ_IMAGE_PATH $IPXE_PATH/$BZ_IMAGE_FILENAME
```

To create relative paths

Reply:
Fixed: 8f96ee28cd11aaff6ba336236cb10d971201f0d9